core/MySigning.cpp

```diff
@@ -200,10 +206,11 @@ bool signerSignMsg(MyMessage &msg)
 			_signingNonceStatus=SIGN_WAITING_FOR_NONCE;
 			if (!_sendRoute(build(_msgSign, msg.destination, msg.sensor, C_INTERNAL,
 			                      I_NONCE_REQUEST).set(""))) {
-				SIGN_DEBUG(PSTR("Failed to transmit nonce request!\n"));
+				SIGN_DEBUG(PSTR("!SGN:SGN:NCE REQ TO=%d\n"),
+				           msg.destination); //Failed to transmit nonce request!
 				ret = false;
 			} else {
-				SIGN_DEBUG(PSTR("Nonce requested from %d. Waiting...\n"), msg.destination);
+				SIGN_DEBUG(PSTR("SGN:SGN:NCE REQ TO=%d\n"), msg.destination); //Nonce requested
```

Comment:
Well, I am quite uncertain in how to be consistent in this matter.
Ex: https://github.com/mysensors/MySensors/blob/development/core/MyTransport.cpp#L759
Perhaps chaing to SIGN_DEBUG(PSTR("SGN:SGN:NCE REQ,TO=%d\n"), msg.destination); //Nonce requested
in applicable places?